tasks/drop.lasso.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: "drop.lasso: Install periodic script"
+  ansible.builtin.template:
+    dest: /usr/local/etc/periodic/daily/900.drop-lasso
+    src: "{{ pf_drop_lasso_template }}"
+    owner: root
+    group: wheel
+    mode: "0755"
+    # keeping a backup will result in the script being run multiple times
+    backup: no
+  when: pf_drop_lasso_enable | bool
+  
+- name: "drop.lasso: Configure periodic."
+  community.general.sysrc:
+    path: "{{ pf_drop_lasso_periodic_file }}"
+    name: "{{ item.name }}"
+    value: "{{ item.value }}"
+  loop: "{{ pf_drop_lasso_rcconf }}"
```

Comment:
This solution always sets the `filename` values. I wanted to avoid setting them unless the user specified them.

```
[13:54 gw01 dvl /etc] % cat /etc/periodic.conf.local   
daily_pf_droplasso_enable="True"
daily_pf_droplasso_v4="YES"
daily_pf_droplasso_v6="YES"
daily_pf_droplasso_v4_filename="/etc/pf.drop_v4.txt"
daily_pf_droplasso_v6_filename="/etc/pf.drop_v6.txt"
```
This is only a nice thing. Not a requirment.

Reply:
I see. You can remove the *.filename defaults from the template. But, *daily_pf_droplasso_enable* should be fixed to YES/NO.